### PR TITLE
Add tests for workflowproj/operator.go

### PR DESCRIPTION
**Pull request by rgolangh**
Signed-off-by: Roy Golan <rgolan@redhat.com>

### CODE REVIEW

**Comment by rgolangh** [comment]
@wmedvede more coverage for the lables work

**Comment by wmedvede** [comment]
@rgolangh looks like make generate-all is required here maybe?

**Comment by rgolangh** [comment]
> @rgolangh looks like make generate-all is required here maybe?

yes forgot that, thanks. Fixing

File: workflowproj/operator_test.go [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+func TestCreateNewUserPropsConfigMap(t *testing.T) {
+	type args struct {
+		workflow *operatorapi.SonataFlow
+	}
+	tests := []struct {
+		name string
+		args args
+		want *corev1.ConfigMap
+	}{
+		//TODO add tests
```
Comment (ricardozanini) [inline workflowproj/operator_test.go:89]: missing tests?

**Review by ricardozanini — COMMENTED** [review]
@rgolangh do you think you can get back to this PR?

**Review by ricardozanini — APPROVED** [review]

**Review by wmedvede — APPROVED** [review]

**Comment by ricardozanini** [comment]
@rgolangh cau you please rebase?

**Pull request merged**